interchaintest/chain_upgrade_test.go

```
@@ -0,0 +1,206 @@
+)
+
+var (
+	// baseChain is the current version of the chain that will be upgraded from
+	baseChain = ibc.DockerImage{
+		Repository: "ghcr.io/liftedinit/manifest-ledger", // GitHub Container Registry path
+		Version:    "v0.0.1-rc.4",                        // The version we're upgrading from
+		UIDGID:     "1025:1025",
+	}
+
+	// Initialize group policy with decision policy
+	_ = func() error {
```

golangci-lint (1.62.2)

init$1 - result 0 (error) is always nil

(unparam)